src/store/index.js

```diff
@@ -131,6 +136,19 @@ const reducer = (state, action) => {
                 state.isMouseEventsEnabled = true;
             }
             break;
+        case 'ENABLE_TRACKED_EVENTS':
+            state.trackedEvents.isActive = action.payload;
+            break;
+        case 'ADD_TRACKED_EVENT':
+            if (!state.trackedEvents.isActive) {
+                return state;
```

Comment:
It would be but If tomorrow the code below the switch evolve it could be an issue. Plus, it's a dirty tricks to avoid `log.debug('Store updated below type - payload - result', action.type, action.payload, state);` at the end of the function.